src/Microsoft.ML.LightGbm/LightGbmRankingTrainer.cs

```diff
@@ -130,8 +155,8 @@ static Options()
             {
                 var res = base.ToDictionary(host);
                 res[GetOptionName(nameof(Sigmoid))] = Sigmoid;
-                res[GetOptionName(nameof(CustomGains))] = string.Join(",",CustomGains);
-                if(EvaluationMetric != EvaluateMetricType.Default)
+                res[GetOptionName(nameof(CustomGains))] = string.Join(",", CustomGains);
+                if (EvaluationMetric != EvaluateMetricType.Default)
                     res[GetOptionName(nameof(EvaluateMetricType))] = GetOptionName(EvaluationMetric.ToString());
```

Comment:
please don't change code at this stage, even if it's trivial. we want to make it easy for Shauheen to take all the changes without any second thought.

Reply:
this just a formating thing, no code just. The underlying jit code and assembly code will be identical, I promise. I'd even bet my paycheck on this ... :)

Reply:
Yep, I'm with @shmoradims. It will be easier to remove these changes even if they are just formatting